docs/structure.md

```diff
@@ -48,7 +48,7 @@ The GreenX library [@GitHub;@azizi_minimax] will eventually provide a variety of
 
 GreenX is written in Fortran 2008. Functionality needed for testing and error handling is written in C and Python. We utilize modern Fortran features such as object-oriented programming and intrinsic procedures that are available in Fortran 2008. We have developed a clear interface between the module code (our library) and the client code (MBPT code), promoting better modularity and reusability. Additionally, we use allocatable arrays and automatic deallocation to simplify the code and avoid memory-related issues, such as leaks and dangling pointers. The implementation is robust and reliable, as we use error handling techniques to highlight and recover from exceptions.
```

Comment:
also @Panadestein and @fdelesma this could either go to the general description of the library or we change the title from Structure of the library to something else

Reply:
@aziziph: I agreed with @dgolze to move this paragraph to the  Overview section in the main page.